lib/internal/http2/compat.js

```diff
@@ -4,9 +4,8 @@ const { Object, ObjectPrototype, Reflect } = primordials;
 
 const assert = require('internal/assert');
 const Stream = require('stream');
-const Readable = Stream.Readable;
-const binding = internalBinding('http2');
-const constants = binding.constants;
+const { Readable } = Stream;
```

Comment:
```suggestion
const { Readable } = require('stream');
```

Reply:
thing is that `Http2ServerResponse` extends from `Stream` below. so if w could use ES6 modules internally, I'd use a 'default', and a  'named import'.